install_config/aggregate_logging.adoc

```diff
@@ -1139,20 +1081,13 @@ $ oc rollout latest dc/logging-curator-ops
 [[aggregate-logging-cleanup]]
 == Cleanup
 
-Remove everything generated during the deployment while leaving other project
-contents intact:
+Remove everything generated during the deployment
 
 ----
-$ oc new-app logging-deployer-template --param MODE=uninstall
+$ ansible-playbook playbooks/common/openshift-cluster/openshift_logging.yml \
+    -e openshift_logging_install_logging=False
 ----
 
-[[aggregate-logging-upgrading]]
-== Upgrading
-
-To upgrade the EFK logging stack, see
-xref:../install_config/upgrading/manual_upgrades.adoc#manual-upgrading-efk-logging-stack[Manual
-Upgrades].
-
 [[troubleshooting-kibana]]
 == Troubleshooting Kibana
 
```

Comment:
@richm @ewolinetz   Not reflected specifically at this line, but this section regarding troubleshooting basically says 'modify THIS value and rerun the deployer.'  What should be our message going forward with troubleshooting and ansible be.  Thoughts?  'Check for XXXXX, update inventory, rerun playbook...'